examples/lusid/corporate-actions-source/create-corporate-action-source.sql

```diff
@@ -0,0 +1,22 @@
+-- ===============================================================
+-- Description:
+-- In this query, we write a corporate action source to LUSID.
+-- ===============================================================
+
+-- Defining corporate action scope and code.
+
+@@corporateActionScope = select 'luminesce-examples';
+@@corporateActionCode = select 'uk-equity';
+
+-- Define a table containing sample information.
+@corporate_action_source_table = 
+
+select  "Sample Actions Source" as DisplayName,
+        @@corporateActionScope as CorporateActionSourceScope,
+        @@corporateActionCode as CorporateActionSourceCode,
+        @@corporateActionScope as InstrumentScope,
+        #2000-01-01# as AsAt;
```

Comment:
The columns I would include here are CorporateActionSourceScope, CorporateActionSourceCode, DisplayName and WriteAction (Insert or Delete). Description is optional but I would also include it.